Sources/PublishCLICore/ProjectGenerator.swift

```diff
@@ -19,7 +19,7 @@ internal struct ProjectGenerator {
         self.folder = folder
         self.publishRepositoryURL = publishRepositoryURL
         self.publishVersion = publishVersion
-        self.siteName = String(folder.name.capitalized.filter { $0.isLetter })
+        self.siteName = Self.makeSiteName(folderName: folder.name)
```

Comment:
I think this could be implemented in a slightly simpler way, by extending `String` with a `asSiteName()` method that does the following:
```swift
private extension String {
    func asSiteName() -> Self {
        let name = prefix(1).capitalized + dropFirst()
        return name.filter { $0.isLetter }
    }
}
```
Then the assignment here could simply be: `self.siteName = folder.name.asSiteName()`.

Reply:
Yes, this implementation is better. I use it in the second commit. But I slightly changed your variant to handle cases like "1-website".